src/Microsoft.Restier.Core/Conventions/ConventionalChangeSetConstants.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+namespace Microsoft.Restier.Core.Conventions
+{
+    internal static class ConventionalChangeSetConstants
+    {
+#region Authorizing method constants
+        public const string AuthorizeMethodNamePrefix = "Can";
+
+        public const string AuthorizeMethodDataModificationInsert = "Insert";
+
+        public const string AuthorizeMethodDataModificationUpdate = "Update";
+
+        public const string AuthorizeMethodDataModificationDelete = "Delete";
+
+        public const string AuthorizeMethodActionInvocationExecute = "Execute";
+#endregion
+
+#region Filtering method constants
+        public const string FilterMethodNamePrefix = "On";
```

Comment:
consider remove this made other OnInsert OnUpdat, OnDelet? Also for the previous Can